src/shared/components/cells/CellContext.tsx

```diff
@@ -42,11 +55,19 @@ const CellContext: FC<Props> = ({
   history,
   location,
   cell,
+  variables,
   onCloneCell,
   onDeleteCell,
   onCSVDownload,
   onRefresh,
 }) => {
+  const viewWithQueries = view as View<
+    Exclude<ViewProperties, MarkdownViewProperties>
+  >
+  const foundQueries =
+    Array.isArray(viewWithQueries?.properties?.queries) &&
```

Comment:
reference `view` instead of `viewWithQueries` here

Reply:
this is view, just properly typed to avoid the markdown stuff, which has no queries property